ui/app/helpers/utils/util.js

```diff
@@ -213,7 +213,27 @@ export function getContractAtAddress (tokenAddress) {
   return global.eth.contract(abi).at(tokenAddress)
 }
 
+export function getRandomFileName () {
+  let fileName = ''
+  const charBank = [
+    ...'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789',
+  ]
+  const fileNameLength = Math.floor((Math.random() * 7) + 6)
+
+  for (let i = 0; i < fileNameLength; i++) {
+    fileName += charBank[
+      Math.floor(Math.random() * charBank.length)
```

Comment:
Nit: This could maybe go in one line, this reads as if this is 1 of many arguments but there's only 1 index

Reply:
Sure I can make that change, I just typically do this formatting when I have any logic inside an index accessor

Reply:
done